### PR TITLE
Add Mandarin word for Czechia to Czech reacts

### PR DESCRIPTION
**Pull request by milspectacular**
Um yes hi

### CODE REVIEW

**Review by howardt12345 — COMMENTED** [review]
There should be a test file for the Czech matching, add this case to it and then you'd be good

File: tests/test_reacts/test_reacts_cz.py [hunk 1]
```diff
@@ -86,11 +83,13 @@
     "Tajwansky",
     "Tajwanec",
     "Tajwanka",
+    # Added Mandarin variant
```
Comment (howardt12345) [inline tests/test_reacts/test_reacts_cz.py:86]:
```suggestion
    # Mandarin variant
```

File: tests/test_reacts/test_reacts_cz.py [hunk 2]
```diff
@@ -105,9 +104,13 @@ def test_react_czech_regex_yes_match(test_str: str):
...
 @pytest.mark.parametrize(
     "test_str",
-    POSTITIVE_TEST_CASES,
+    POSITIVE_TEST_CASES,
 )
 def test_react_czech_regex_no_match(test_str: str):
     """Tests that these strings return FALSE."""
-    # * surrounded by text
-    assert not check_resource_match(f"a{test_str}b", resource_name='czech')
+    if test_str == "捷克":
```
Comment (howardt12345) [inline tests/test_reacts/test_reacts_cz.py:111]: Conflicted on hardcoding the text string, not ideal imo 
IIRC the Ukrainian test has mandarin handling, check out how it's done in there instead

File: tests/test_reacts/test_reacts_cz.py
Comment (chanomkaimuk) [inline tests/test_reacts/test_reacts_cz.py:1]: why are all the `#variant` comments removed lol (though it doesn't change anything test-wise)

**Review by chanomkaimuk — APPROVED** [review]
lgtm (looks good to meow)

File: tests/test_reacts/test_reacts_cz.py [hunk 3]
```diff
@@ -105,9 +106,23 @@ def test_react_czech_regex_yes_match(test_str: str):
...
+)
+def test_react_czech_mandarin_yes_match(test_str: str):
+    """Tests that these Mandarin strings return TRUE."""
+    # * isolated string
+    assert check_resource_match(test_str, resource_name='czech')
+    # * surrounded by spaces
+    assert check_resource_match(f" {test_str} ", resource_name='czech')
+    # * surrounded by text (ok for Mandarin)
+    assert check_resource_match(f"a{test_str}b", resource_name='czech')
+    assert check_resource_match(f"哈{test_str}囉", resource_name='czech')
```
Comment (howardt12345) [inline tests/test_reacts/test_reacts_cz.py:128]: This should fail

**Review by howardt12345 — CHANGES_REQUESTED** [review]

**Review by howardt12345 — APPROVED** [review]

**Pull request merged**